telemetry/about-telemetry.adoc

```diff
@@ -0,0 +1,17 @@
+:_content-type: ASSEMBLY
+[id="about-telemetry"]
+= About Telemetry
+include::modules/common-attributes.adoc[]
+:context: about-telemetry
+
+toc::[]
+
+
+[role="_abstract"]
+{rh-rhacs-first} collects anonymized aggregated information about product usage and product configurations.
+This information can help Red Hat to better understand how customers use the product and which areas to prioritize for improvements.
+
+Red Hat uses this information to improve the user experience. None of the information is shared with third parties.
```

Comment:
Not sure about the information sharing sentence.

Reply:
Removing the information sharing sentence.